<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>factory</class>
 <widget class="QMainWindow" name="factory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Factory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="qfLeftZone">
      <property name="minimumSize">
       <size>
        <width>252</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>252</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="vlLeftZone">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="qfSwitch">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QHBoxLayout" name="hlSwitch">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbTexture">
            <property name="text">
             <string>Texture</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bgSwitch</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbShader">
            <property name="text">
             <string>Shader</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bgSwitch</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbStockShot">
            <property name="text">
             <string>StockShot</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bgSwitch</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="twTree">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="qfRightZone">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="vlRightZone">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="twTexture">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="indentation">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">Stored Textures</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
         </widget>
         <widget class="QTreeWidget" name="twShader">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="indentation">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">Stored Shaders</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
         </widget>
         <widget class="QTreeWidget" name="twStockShot">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="indentation">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string notr="true">Stored StockShots</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeWidget" name="twThumbnail">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">5</string>
        </property>
       </column>
      </widget>
      <widget class="QFrame" name="qfInfo">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>205</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSpinBox" name="sbColumns">
            <property name="prefix">
             <string>Columns = </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbStorage">
            <property name="text">
             <string>Storage</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="teInfo">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mThumbnail">
    <property name="title">
     <string>Thumbnail</string>
    </property>
    <addaction name="miCreateSelPreviewFiles"/>
    <addaction name="miCreateAllPreviewFiles"/>
    <addaction name="separator"/>
    <addaction name="miCreateSelMovies"/>
    <addaction name="miCreateAllMovies"/>
    <addaction name="separator"/>
    <addaction name="miSaveShader"/>
   </widget>
   <widget class="QMenu" name="mTransfert">
    <property name="title">
     <string>Transfert</string>
    </property>
    <addaction name="miClearTexture"/>
    <addaction name="miClearShader"/>
    <addaction name="miClearStockShot"/>
    <addaction name="separator"/>
    <addaction name="miStoredTexture"/>
    <addaction name="miStoredShader"/>
    <addaction name="miStoredStockShot"/>
   </widget>
   <addaction name="mTransfert"/>
   <addaction name="mThumbnail"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="miCreateSelIcons">
   <property name="text">
    <string>Create Selected Icons</string>
   </property>
  </action>
  <action name="miCreateAllIcons">
   <property name="text">
    <string>Create All Icons</string>
   </property>
  </action>
  <action name="miCreateSelPreviews">
   <property name="text">
    <string>Create Selected Previews</string>
   </property>
  </action>
  <action name="miCreateAllPreviews">
   <property name="text">
    <string>Create All Previews</string>
   </property>
  </action>
  <action name="miStoredTexture">
   <property name="text">
    <string>Stored Texture</string>
   </property>
  </action>
  <action name="miStoredShader">
   <property name="text">
    <string>Stored Shader</string>
   </property>
  </action>
  <action name="miStoredStockShot">
   <property name="text">
    <string>Stored StockShot</string>
   </property>
  </action>
  <action name="miCreateSelPreviewFiles">
   <property name="text">
    <string>Create Selected Preview Files</string>
   </property>
  </action>
  <action name="miCreateAllPreviewFiles">
   <property name="text">
    <string>Create All Preview Files</string>
   </property>
  </action>
  <action name="miCreateSelMovies">
   <property name="text">
    <string>Create Selected Movies</string>
   </property>
  </action>
  <action name="miCreateAllMovies">
   <property name="text">
    <string>Create All Movies</string>
   </property>
  </action>
  <action name="miClearTexture">
   <property name="text">
    <string>Clear Texture</string>
   </property>
  </action>
  <action name="miClearShader">
   <property name="text">
    <string>Clear Shader</string>
   </property>
  </action>
  <action name="miClearStockShot">
   <property name="text">
    <string>Clear StockShot</string>
   </property>
  </action>
  <action name="miSaveShader">
   <property name="text">
    <string>Save Shader</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bgSwitch"/>
 </buttongroups>
</ui>
